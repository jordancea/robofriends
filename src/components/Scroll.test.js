import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import Scroll from "./Scroll.js";

it("should render the Scroll wrapper component", () => {
  const mochSearchChange = jest.fn();
  const wrapper = shallow(
    <Scroll>
      <div>Test</div>
    </Scroll>
  );
  expect(toJson(wrapper)).toMatchSnapshot();
});
