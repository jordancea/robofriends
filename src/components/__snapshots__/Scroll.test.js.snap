// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the Scroll wrapper component 1`] = `
<div
  style={
    Object {
      "border": "5px solid black",
      "height": "800px",
      "overflow": "scroll",
    }
  }
>
  <div>
    Test
  </div>
</div>
`;
